src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -459,6 +464,298 @@ public Color addValuesToHsbComponents(Color baseColor,
   }
 
   /**
+   * Saturate the specified color. First argument is the color, second is the
+   * amount of saturation (from 0 to 100)
```

Comment:
Is that 0-100% more than the current saturation? Or will 0 essentially make it grey? I think you should be explicit in the docs, if you can be.